pydantic/main.py

```diff
@@ -698,7 +698,7 @@ def __getattr__(self, item: str) -> Any:
     def __setattr__(self, name: str, value: Any) -> None:
         if name in self.__class_vars__:
             raise AttributeError(
-                f'"{name}" is a ClassVar of `{self.__class__.__name__}` and cannot be set on an instance. '
+                f'{name!r} is a ClassVar of `{self.__class__.__name__}` and cannot be set on an instance. '
```

Comment:
Non-related change that I talked to @dmontagu on a call. Shouldn't be a big deal... 👀